sw/device/tock/README

```diff
@@ -0,0 +1,43 @@
+Tock OS
+=======
+
+Tock OS is a secure embedded operating system for microcontrollers
+
+Documentation available at:
+https://www.tockos.org/
+https://github.com/tock/tock
+
+Rationale
+=========
+Upstream OpenTitan implementation is not using DIFs, which means that the
+peripheral drivers cannot be directly implemented upstream. Therefore it is not
+possible to use upstream OpenTitan board and chip implementations.
+
+The intention is, where possible, to use upstream packages. At the moment the
+only excpetions are board and chip packages, which are implemented and
+maintained downstream.
+
+Copyright
+=========
+Any files taken directly from the upstream must keep their original copyright.
+
+Any new files must have lowRISC copyright, please see `COPYRIGHT`.
```

Comment:
This section is a bit strange. Can you rewrite it as:

> The source files in this directory come from two sources, with slightly different copyright licences.
>
> Some source files are copied from the upstream tock repository. The notice in `COPYRIGHT` in this directory applies to all source files with explicit Tock OS Developers copyright headers, or no copyright headers at all.
>
> Where lowRISC has added additional files, these are marked with the lowRISC Copyright header, and the notice in `LICENCE` in the root of this repository applies.

Reply:
Done.